README.md

```diff
@@ -8,220 +8,10 @@
-
-The library compares the user from the route and decides upon each one of these parameters and acts accordingly. The developer is told what is happening behind the scenes via `NSNotification`.
-
-### Guidance Notifications
-
-This library relies heavily on `NSNotification`s for letting the developer know when events have occurred.
-
-#### `RouteControllerProgressDidChange`
-
-* Emitted when the user moves along the route. Notification contains 3 keys:
-  * `RouteControllerProgressDidChangeNotificationProgressKey` - `RouteProgress` - Current progress along route
-  * `RouteControllerProgressDidChangeNotificationLocationKey` - `CLLocation` - Current location
-  * `RouteControllerProgressDidChangeNotificationSecondsRemainingOnStepKey` - `Double` - Given users speed and location, this is the number of seconds left to the end of the step
-
-#### `RouteControllerAlertLevelDidChange`
-
-* Emitted when the alert level changes. This indicates the user should be notified about the upcoming maneuver. See [Alerts](#Alert levels). Notification contains 3 keys:
-  * `RouteControllerProgressDidChangeNotificationProgressKey` - `RouteProgress` - Current progress along route
-  * `RouteControllerAlertLevelDidChangeNotificationDistanceToEndOfManeuverKey` - `CLLocationDistance` - The users snapped distance to the end of the route.
-
-#### `RouteControllerShouldReroute`
-
-* Emitted when the user is off the route and should be rerouted. Notification contains 1 key:
-  * `RouteControllerNotificationShouldRerouteKey` - `CLLocation` - Last location of user
-
-### Alert levels
-
-Alert levels indicate the type of announcement that should be given. The enum types available are:
-
-* `none`
-* `depart` - Emitted while departing origin
-* `low` - Emitted directly after completing the maneuver
-* `medium` - Emitted when the user has [70 seconds](https://github.com/mapbox/mapbox-navigation-ios/blob/19365cdad5f18641579a560dfc7113057b3053ad/MapboxNavigation/Constants.swift#L15) remaining on the route.
-* `high` - Emitted when the user has [15 seconds](https://github.com/mapbox/mapbox-navigation-ios/blob/19365cdad5f18641579a560dfc7113057b3053ad/MapboxNavigation/Constants.swift#L16) remaining on the route.
-* `arrive` - Emitted when the user arrives at destination
-
-### Rerouting
-
-In the event of a reroute, it's necessary to update the current route with a new route. Once fetched, you can update the current route by:
-
-```swift
-var navigation: RouteController!
-navigation.routeProgress = RouteProgress(route: newRoute)
-```
+* ### [Documentation](https://www.mapbox.com/mapbox-navigation-ios/navigation/)
+* ### [Installation](https://www.mapbox.com/mapbox-navigation-ios/navigation/0.6.0/installation.html)
+* ### [Getting Started](https://www.mapbox.com/mapbox-navigation-ios/navigation/0.6.0/installation.html)
+* ### [Running the example project](https://www.mapbox.com/mapbox-navigation-ios/navigation/0.6.0/run-example-app.html)
```

Comment:
Will we have to update all the version numbers every time we release? Maybe link just once to the docset’s front page and add “(Installation, getting started, …)” after the link, or duplicate the most basic information here, as @ericrwolfe suggests.